modules/openapi-generator/src/main/resources/python-experimental/api_client.handlebars

```diff
@@ -889,18 +911,51 @@ class OpenApiResponse(JSONDetector):
             return response.data
 
     @staticmethod
-    def __deserialize_multipart_form_data(
-        response: urllib3.HTTPResponse
-    ) -> typing.Dict[str, typing.Any]:
-        msg = email.message_from_bytes(response.data)
-        return {
-            part.get_param("name", header="Content-Disposition"): part.get_payload(
-                decode=True
-            ).decode(part.get_content_charset())
-            if part.get_content_charset()
-            else part.get_payload()
-            for part in msg.get_payload()
-        }
+    def validate(content_type: str) -> bool:
+        return content_type == 'application/octet-stream'
+
+
+class OpenApiResponse(JSONDetector):
+    __deserialization_strategys: typing.Sequence[
+        DeserializationStrategy
+    ] = [
+        JsonDeserializationStrategy,
+        FormDataDeserializationStrategy,
+        OctetDtreamDeserializationStrategy,
+    ]
+    def __init__(
+        self,
+        response_cls: typing.Type[ApiResponse] = ApiResponse,
+        content: typing.Optional[typing.Dict[str, MediaType]] = None,
+        headers: typing.Optional[typing.List[HeaderParameter]] = None,
+    ):
+        self.headers = headers
+        if content is not None and len(content) == 0:
+            raise ValueError('Invalid value for content, the content dict must have >= 1 entry')
+        self.content = content
+        self.response_cls = response_cls
+
+    @classmethod
+    def detect_deserialization_strategy_by_content_type(
+        cls,
+        content_type: str
+    ) -> typing.Optional[DeserializationStrategy]:
+        for strategy in cls.__deserialization_strategys:
+            if strategy.validate(content_type):
+                return strategy
+        return None
+
+    def __ensure_body_schema(self, content_type: str, status_code: int)-> typing.Optional[typing.Type[Schema]]:
+        # some specs do not define response content media type schemas
+        if self.content is not None:
+            for content in self.content:
+                if content_type.startswith(content):
+                    return self.content[content].schema
+            raise ApiValueError(
+                f'Invalid content_type={content_type} returned for response with '
+                f'status_code={str(status_code)}'
```

Comment:
How about something like this?
`Invalid content_type returned for response with status_code=200. content_type='blah' was returned when only {'abc', 'def'} are allowed`